### PR TITLE
fix(aktualizr): use CONFIG_FILE reference

### PR DESCRIPTION
**Pull request by StealthyCoder**
Just use the full CONFIG_FILE path everywhere to make sure we always target it correctly.

### CODE REVIEW

**Review by vkhoroz — APPROVED** [review]
Nice!

Once this is merged, please, also create a PR in meta-lmp to update the git hash, and wait for Ricardo to merge it.

**Review by mike-sul — APPROVED** [review]

**Review by doanac — COMMENTED** [review]
Just to add another layer of protection here. Can we do one of two things near the top:

## an assertion
```
[ -z "$SOTA_DIR" ] && (echo "ERROR: SOTA_DIR not defined"; exit 1)
```
## a 99% safe default
```
if [ -z "$SOTA_DIR" ]; then
    SOTA_DIR="/var/sota"
    echo "WARNING: SOTA_DIR not defined, defaulting to $SOTA_DIR"
fi
```

**Pull request merged**
